boom! testing hashicorp_vault-0.6.1 against nightly for nightly
boom! running: cargo +nightly build --frozen
boom! creating container for: cargo +nightly build --frozen
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/nightly:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/nightly:/target:rw -e USER_ID=500 -e CMD=cargo +nightly build --frozen cargobomb`
blam! a404537f3a5b121c19b17772f02fc97e1572fa88e6ad2f40489ac62515bd0816
boom! running `docker start -a a404537f3a5b121c19b17772f02fc97e1572fa88e6ad2f40489ac62515bd0816`
kablam!    Compiling hashicorp_vault v0.6.1 (file:///source)
kablam!     Finished dev [unoptimized + debuginfo] target(s) in 4.21 secs
boom! running `docker rm -f a404537f3a5b121c19b17772f02fc97e1572fa88e6ad2f40489ac62515bd0816`
blam! a404537f3a5b121c19b17772f02fc97e1572fa88e6ad2f40489ac62515bd0816
boom! running: cargo +nightly test --frozen --no-run
boom! creating container for: cargo +nightly test --frozen --no-run
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/nightly:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/nightly:/target:rw -e USER_ID=500 -e CMD=cargo +nightly test --frozen --no-run cargobomb`
blam! af4d7060a31dcc219343aef4933a174256677a78a9d81d0bd5e244bfc57f093a
boom! running `docker start -a af4d7060a31dcc219343aef4933a174256677a78a9d81d0bd5e244bfc57f093a`
kablam!    Compiling hashicorp_vault v0.6.1 (file:///source)
kablam!     Finished dev [unoptimized + debuginfo] target(s) in 6.20 secs
boom! running `docker rm -f af4d7060a31dcc219343aef4933a174256677a78a9d81d0bd5e244bfc57f093a`
blam! af4d7060a31dcc219343aef4933a174256677a78a9d81d0bd5e244bfc57f093a
boom! running: cargo +nightly test --frozen
boom! creating container for: cargo +nightly test --frozen
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/nightly:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/nightly:/target:rw -e USER_ID=500 -e CMD=cargo +nightly test --frozen cargobomb`
blam! e8b2a34ec31af2194f9c9e108ec8873b7e48052ebc55825a2d10b79b74798f6e
boom! running `docker start -a e8b2a34ec31af2194f9c9e108ec8873b7e48052ebc55825a2d10b79b74798f6e`
kablam!     Finished dev [unoptimized + debuginfo] target(s) in 0.0 secs
kablam!      Running /target/debug/deps/hashicorp_vault-c34e38603fc9d060
blam! 
blam! running 5 tests
blam! test tests::it_can_create_a_client ... FAILED
blam! test tests::it_can_write_secrets_with_newline ... FAILED
blam! test tests::it_can_query_secrets ... FAILED
blam! test tests::it_can_delete_a_secret ... FAILED
blam! test tests::it_returns_err_on_forbidden ... ok
blam! 
blam! failures:
blam! 
blam! ---- tests::it_can_create_a_client stdout ----
blam! 	thread 'tests::it_can_create_a_client' panicked at 'called `Result::unwrap()` on an `Err` value: Hyper(Io(Error { repr: Os { code: 111, message: "Connection refused" } }))', /checkout/src/libcore/result.rs:859
blam! note: Run with `RUST_BACKTRACE=1` for a backtrace.
blam! 
blam! ---- tests::it_can_write_secrets_with_newline stdout ----
blam! 	thread 'tests::it_can_write_secrets_with_newline' panicked at 'called `Result::unwrap()` on an `Err` value: Hyper(Io(Error { repr: Os { code: 111, message: "Connection refused" } }))', /checkout/src/libcore/result.rs:859
blam! 
blam! ---- tests::it_can_query_secrets stdout ----
blam! 	thread 'tests::it_can_query_secrets' panicked at 'called `Result::unwrap()` on an `Err` value: Hyper(Io(Error { repr: Os { code: 111, message: "Connection refused" } }))', /checkout/src/libcore/result.rs:859
blam! 
blam! ---- tests::it_can_delete_a_secret stdout ----
blam! 	thread 'tests::it_can_delete_a_secret' panicked at 'called `Result::unwrap()` on an `Err` value: Hyper(Io(Error { repr: Os { code: 111, message: "Connection refused" } }))', /checkout/src/libcore/result.rs:859
blam! 
blam! 
blam! failures:
blam!     tests::it_can_create_a_client
blam!     tests::it_can_delete_a_secret
blam!     tests::it_can_query_secrets
blam!     tests::it_can_write_secrets_with_newline
blam! 
blam! test result: FAILED. 1 passed; 4 failed; 0 ignored; 0 measured
blam! 
kablam! error: test failed
boom! running `docker rm -f e8b2a34ec31af2194f9c9e108ec8873b7e48052ebc55825a2d10b79b74798f6e`
blam! e8b2a34ec31af2194f9c9e108ec8873b7e48052ebc55825a2d10b79b74798f6e
